core/proto/squareup/subzero/internal.proto

```diff
@@ -26,11 +26,13 @@ message InternalCommandRequest {
   message SignTxRequest {
     required EncryptedMasterSeed encrypted_master_seed = 1;
     repeated EncryptedPubKey encrypted_pub_keys = 2;
+    //deprecate these fields.
```

Comment:
I like this note to our future selves! It'll make our life certainly easier. Do you think we should refer to a specific ticket, and add a comment in the places we'll have to modify (so that it's easier to grep when we come back to it)?

Reply:
good point. I am adding some comments accross the code.
Also i would probably not put JIRA tickets here as this is public.